samples/client/petstore/cpp-restsdk/client/CMakeLists.txt

```diff
@@ -8,16 +8,29 @@
 #
 # NOTE: Auto generated by OpenAPI Generator (https://openapi-generator.tech).
 
-cmake_minimum_required (VERSION 3.1)
+cmake_minimum_required (VERSION 3.5)
 
-project(CppRestPetstoreClient)
+project(CppRestPetstoreClient CXX)
 
 # Force -fPIC even if the project is configured for building a static library.
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
-set(CXX_STANDARD_REQUIRED ON)
 
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
@aminya did you need to add the above line to CMakeLists file in order to build the client?

I tried the above but no luck.

Reply:
https://github.com/OpenAPITools/openapi-generator/pull/18821#discussion_r1623345714